### PR TITLE
Edits based on Peer review

### PR DESCRIPTION
**Pull request by Hussein-Mahfouz**
I have addressed most of the edits received by the reviewers, and a "response to reviewers" document has also been added. . Two things that haven't been properly addressed:

1. We were asked for a **Schematization of the methodology**. Do I need to add a flowchart for the methodology
2. "The big picture take aways from the work were also not clear…". Thoughts on where we should make changes?

@Robinlovelace you mentioned that we can create a diff in the PDF file, similar to tracked changes in word. Any pointers on how to do that?

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -11,15 +11,15 @@ author:
     institute: its
 institute:
   - casa: Centre for Advanced Spatial Analysis (CASA), University College London, London, UK
-  - its: Institute of Transport Studies, University of Leeds, Leeds, UK
+  - its: Institute for Transport Studies, University of Leeds, Leeds, UK
```
Comment (Robinlovelace) [inline README.Rmd:14]: :

File: README.Rmd [hunk 2]
```diff
@@ -117,7 +125,7 @@ area's bicycle network denser means adding more cycling routes in the
 area and thereby giving cyclists more route options.] of the cycling
 network is also vital [@schoner2014missing].
 
-## Network-Level Approaches
+## Network-Level Approaches to Bicycle Infrastructure Planning
```
Comment (Robinlovelace) [inline README.Rmd:128]: :+1:

File: README.Rmd [hunk 3]
```diff
@@ -153,16 +161,15 @@ improving roadway links to meet a desired level of service (LOS)
...
 that each link with a bike lane should be connected to at least one
 destination [@mesbah2012bilevel], a constraint on maximum deviation
 from shortest paths [@duthie2014optimization], or a discontinuity
 penalty to prioritize connected road segments [@mauttone2017bicycle].
 
 In this paper, continuity is analysed by looking at the connectivity of
 the network through the graph-theoretic concept of *connected
-components*. @natera2019data for example,
-study the existing cycling network in terms of its disconnected
+components*. @orozco2020data study the existing cycling network in terms of its disconnected
```
Comment (Robinlovelace) [inline README.Rmd:172]: :+1:

**Review by Robinlovelace — APPROVED** [review]
This looks great. Re diff, I will give it a try...

**Comment by Hussein-Mahfouz** [comment]
@Robinlovelace I've made some minor edits to the conclusion and merged to this pull request. Should I go ahead and merge, or does your latex diff script need to run before merging?

**Pull request merged**